gui/resources.xml

```diff
@@ -1670,6 +1673,7 @@ Vocabulary
           <property name="icon-name">mypaint-layer-move-symbolic</property>
           <signal name="activate" handler="mode_flip_action_activated_cb"/>
         </object>
+        <accelerator key="m"/>
```

Comment:
but adding a control modifier to it wouldn't hurt. It would free it up for any future drawing tools modes.